### PR TITLE
Use header indicators on basic profiles info

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR changes the basic info rendering on profiles (actor and places) so we render the indicators coming from the headerAttributes instead of the former ones.

Some changes on the backend might be done in the future to get rid of the NUM_DECIMALS list that has hardcoded decimal numbers for the indicator values.

image

### CODE REVIEW

File: frontend/scripts/react-components/profiles/actor-summary.component.jsx [hunk 1]
```diff
@@ -49,74 +138,14 @@ class ActorSummary extends React.PureComponent {
...
-                      className="stats-text"
-                    >
-                      NONE
-                    </Text>
-                  </div>
-                )}
-              </div>
-            )}
-          </div>
+          {!isEmpty(
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profiles/actor-summary.component.jsx]: isEmpty could be replaced with a few more lines, besides that, using an expression inside isEmpty is weird.

File: frontend/scripts/react-components/profiles/place-summary.component.jsx [hunk 2]
```diff
@@ -49,9 +46,6 @@ function PlaceSummary(props) {
     { name: capitalize(biomeName), label: 'Biome' },
     { name: capitalize(stateName), label: 'State' }
   ];
-  const commodityAreaValue = formatValue(commodityArea, 'area');
-  const areaValue = formatValue(area, 'area');
-  const commodityProductionValue = formatValue(commodityProduction, 'tons');
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profiles/place-summary.component.jsx]: I think that if commodityProduction is tons here, the new value in the constants should also be tons.

File: frontend/scripts/react-components/profiles/place-summary.component.jsx [hunk 3]
```diff
@@ -73,55 +67,28 @@ function PlaceSummary(props) {
   );
 
   const renderStats = () =>
-    (areaValue !== '-' || commodityAreaValue !== '-' || commodityProductionValue !== '-') && (
+    !isEmpty(headerAttributes) &&
```
Comment (sorodrigo) [inline frontend/scripts/react-components/profiles/place-summary.component.jsx]: we can replace isEmpty with a few more lines

**Comment by Bluesmile82** [comment]
Ready again!

**Pull request merged**